web_src/js/features/repo-diff.js

```diff
@@ -89,6 +90,18 @@ export function initRepoDiffConversationNav() {
   });
 }
 
+// Will be called when the show more (files) button has been pressed
+function onShowMoreFiles() {
+  initRepoIssueContentHistory();
+  initViewedCheckboxListenerFor(document);
+
+  // The number of files is constant, but the number of viewed files can change because more files could be viewed
+  /* eslint-disable */
+  window.config.pageData.numberOfViewedFiles = document.querySelectorAll(`${viewedCheckboxSelector} > input[type=checkbox][checked]`).length;
+  /* eslint-enable */
```

Comment:
I've tested it, and the only thing that would work is the following:
```suggestion
  window.config['pageData'].numberOfViewedFiles = document.querySelectorAll(`${viewedCheckboxSelector} > input[type=checkbox][checked]`).length;
```
Which is better? Leaving the comments or appeasing the mighty formatter?
Because to me, this option looks by far less readable than the current solution.
What's your opinion on that?

Reply:
The current line LGTM what about rather changing line 7 for this specific file? If that would be:
`const csrfToken = window.config.csrfToken;`
Because changing this line would only complicate it more IMO(or the eslint-rule somehow can detect that this case is ridiculous)